pythoncode/FortiusAnt.py

```diff
@@ -707,6 +707,16 @@ def Tacx2Dongle(self):
                             if False and clv.PowerMode and debug.on(debug.Application):
                                 logfile.Write('PowerMode: TargetPower info received - timestamp set')
 
+                        #-------------------------------------------------------
+                        # Data page 50 (0x32) Wind resistance
+                        #-------------------------------------------------------
+                        elif DataPageNumber == 50:

```

Comment:
if this page is received, it will overwrite the globals in usbTrainer - if it isn't received, the glabals already have default values.